firebase-config/src/main/java/com/google/firebase/remoteconfig/RemoteConfigComponent.java

```diff
@@ -73,8 +78,11 @@ public class RemoteConfigComponent {
   @GuardedBy("this")
   private final Map<String, FirebaseRemoteConfig> frcNamespaceInstances = new HashMap<>();
 
+  private static final Map<String, FirebaseRemoteConfig> frcNamespaceInstancesStatic =
+      new HashMap<>();
+
   private final Context context;
-  private final Executor executor;
+  private final ScheduledExecutorService executor;
```

Comment:
Question for platform reviewers - is it safe to swap these executor implementations throughout RC?

Reply:
Yeah that should be fine since it's a subclass, although we also have [predefined executors](https://firebase.github.io/firebase-android-sdk/components/executors/) that folks are highly _encouraged_ to use.